### ISSUE TITLE
docker daemon need to start with insecure registry 

### ISSUE THREAD

**Issue by praveenkumar** [jimmidyson/minishift#2]
If we want to work with openshift as developer then inside minishift docker daemon should be running with `--insecure-registry 172.30.0.0/16`. 

```
[prkumar@dhcp193-119 minishift]$ ./out/minishift docker-env
export DOCKER_HOST=tcp://192.168.42.110:2376
export DOCKER_CERT_PATH=/home/prkumar/.minishift/certs
export DOCKER_TLS_VERIFY=1
# Run this command to configure your shell: 
# eval $(minishift docker-env)

[prkumar@dhcp193-119 minishift]$ eval $(./out/minishift docker-env)

[prkumar@dhcp193-119 minishift]$ docker images
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE

[prkumar@dhcp193-119 minishift]$ ./oc cluster up
-- Checking Docker client ... OK
-- Checking for existing OpenShift container ... OK
-- Checking for openshift/origin:latest image ... 
   Pulling image openshift/origin:latest
   Pulled 1/4 layers, 26% complete
   Pulled 2/4 layers, 70% complete
   Pulled 3/4 layers, 81% complete
   Pulled 3/4 layers, 95% complete
   Pulled 4/4 layers, 100% complete
   Extracting
   Image pull complete
-- Checking Docker daemon configuration ... FAIL
   Error: did not detect an --insecure-registry argument on the Docker daemon
   Solution:

     Ensure that the Docker daemon is running with the following argument:
     	--insecure-registry 172.30.0.0/16
```

**Comment by rhuss** [comment]
Why are you using `oc cluster up` here ? `minishift` already comes with OpenShift running, so all you need is `minishift start` to have a single-node OpenShift installation.

**Comment by praveenkumar** [comment]
@rhuss what about routes and registry configuration?

**Comment by jimmidyson** [comment]
Sorry screwed up path for iptables. Wii push simple fix for this in a few hours.

**Comment by jimmidyson** [comment]
Fixed by 8545594844484d5a093beaaa2218dc827932fae7. Released [v0.1.1](https://github.com/jimmidyson/minishift/releases/tag/v0.1.1)
